### DISCUSSION TITLE
Help. win 7

### DISCUSSION THREAD

**Discussion by deleteBlack666** [streamlink/streamlink-twitch-gui#901] [Q&A] [answered]
![12345](https://user-images.githubusercontent.com/78680082/176912607-0eaaffb2-963d-400f-85c0-4a81cc2ed01f.PNG)
Here is the error when trying to watch the broadcast. what to do? I'm a noob hahahahahahahaha

**Comment by bastimeyer** [comment, marked as answer]
https://github.com/streamlink/streamlink-twitch-gui/wiki/Installation
https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration

**Reply by deleteBlack666** [reply]
> https://github.com/streamlink/streamlink-twitch-gui/wiki/Installation https://github.com/streamlink/streamlink-twitch-gui/wiki/Streamlink-configuration
streamlink-4.1.0-3-py310-x86_64.exe is streamlink-4.1.0-3-py38-x86_64.exen what to download from this?

**Reply by bastimeyer** [reply]
Read the docs and readme, it's all described there, and it's not difficult. Just read. I don't have to repeat this here.

https://streamlink.github.io/latest/install.html#windows-binaries
https://github.com/streamlink/windows-builds/releases/tag/4.1.0-3

> Users of Windows 7 need to choose the release files based on Python 3.8.

**Comment by deleteBlack666** [comment]
Everything is working
